cmd/loop/liquidity.go

```diff
@@ -286,6 +286,11 @@ var setParamsCommand = cli.Command{
 				"of swaps, limited to the budget set by " +
 				"autobudget",
 		},
+		cli.StringFlag{
+			Name: "outaddr",
```

Comment:
nit: for the loop command, this same parameter is "--addr", I don't know how much the consistency matters to you